keyboards/sol/config.h

```diff
@@ -20,3 +20,75 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #include "config_common.h"
 
+#define PREVENT_STUCK_MODIFIERS
+#define TAPPING_FORCE_HOLD
```

Comment:
Cries in One Shot

Reply:
this was the default sol behavior, just restructured the directory a bit in prop for r2

Reply:
That's fine.  It just breaks one shots, tap toggle, and some other functionality.

Reply:
I'll ask @Legonut about removing it.